pyServer/manifests/linux/diagnostic

```diff
@@ -104,6 +110,7 @@ copy,/var/log/dmesg*
 copy,/var/log/boot*
 copy,/var/log/auth*
 copy,/var/log/secure*
+copy,/var/log/cloudregister
```

Comment:
How big this file can be?
As well /var/log/zypp/history and /boot ?
Just need to make sure that we are not adding a big chunk as part of diagnostic manifest.
Also for all of these new addition, if you are sure for any of that that it does not contain any secret then we can add "noscan" to skip credscan and improve the performance.